### PR TITLE
fixed error on FASTGRPAVE data

### PR DESCRIPTION
**Pull request by jemorrison**
Fixed problem in #2049

### CODE REVIEW

File: jwst/rscd/rscd_sub.py [hunk 1]
```diff
@@ -217,8 +217,8 @@ def get_rscd_parameters(input_model, rscd_model):
         log.warning('No matching row found in RSCD reference table for')
         log.warning('READPATT=%s, SUBARRAY=%s, Row type=EVEN',
                     readpatt, subarray)
-        raise NoTableEntry('No match for READPATT or SUBARRAY in RSCD ref table')
-        return None, None
+        log.warning('RSCD correction will be skipped, only 1 Group need at least 2')
```
Comment (hbushouse) [inline jwst/rscd/rscd_sub.py]: The last phrase here "only 1 Group need at least 2" is not relevant to this situation. The appropriate reason was already stated in the warning message in line 217. So this message can be trimmed to just "RSCD step will be skipped."

File: jwst/rscd/rscd_sub.py [hunk 2]
```diff
@@ -249,8 +249,8 @@ def get_rscd_parameters(input_model, rscd_model):
         log.warning('No matching row found in RSCD reference table for')
         log.warning('READPATT=%s, SUBARRAY=%s, Row type=ODD',
                     readpatt, subarray)
-        raise NoTableEntry('No match for READPATT or SUBARRAY in RSCD ref table')
-        return None, None
+
```
Comment (hbushouse) [inline jwst/rscd/rscd_sub.py]: Should add the same warning here as in line 220, "RSCD step will be skipped", before returning.

File: jwst/rscd/rscd_sub.py (hunk 1 above)
Comment (jemorrison) [reply jwst/rscd/rscd_sub.py]: it is not rscd_sub that returns NONE but a function in rscd_sub (get_rscd_parameters) that returns NON to rscd_sub. If none is returned from get_rscd_parameters to rscd_sub
then a warning message is sent that the table value could not be found, output.meta.cal_step.rscd is set to skipped and output is returned (output is set to input_model.copy() in the beginning of rscd_sub.

Comment (jemorrison) [reply jwst/rscd/rscd_sub.py]: I took out that extra (incorrect warning message) in the most recent pr

**Comment by hbushouse** [comment]
Testing using a version of the code that removes line 220 I get the following result for the dataset in question:
```
2018-05-22 10:30:15,816 - stpipe.Detector1Pipeline.rscd - INFO - Step rscd running with args (<RampModel(1, 20, 1024, 1032) from jw80600013001_02101_00001_mirimage_uncal.fits>,).
2018-05-22 10:30:15,852 - stpipe.Detector1Pipeline.rscd - INFO - Using RSCD reference file /grp/crds/cache/references/jwst/jwst_miri_rscd_0009.fits
2018-05-22 10:30:16,053 - stpipe.Detector1Pipeline.rscd - WARNING - No matching row found in RSCD reference table for
2018-05-22 10:30:16,053 - stpipe.Detector1Pipeline.rscd - WARNING - READPATT=FASTGRPAVG, SUBARRAY=FULL, Row type=EVEN
2018-05-22 10:30:16,053 - stpipe.Detector1Pipeline.rscd - WARNING - RSCD correction will be skipped
2018-05-22 10:30:16,057 - stpipe.Detector1Pipeline.rscd - INFO - Step rscd done
```
so that looks good.

**Pull request merged**